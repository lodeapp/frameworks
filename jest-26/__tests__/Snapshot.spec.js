describe('A test with snapshots', () => {
    it('Matches snapshot', () => {
        expect('Velit dolore laboris qui est.').toMatchSnapshot()
    })

    it('Matches HTML snapshot', () => {
        const html = `<div>
  <h2>Hello world!</h2>
  <p>Esse excepteur aliqua proident aliquip deserunt fugiat officia et consequat sit sint aliqua exercitation.</p>
  <small>Culpa cupidatat ut velit aliqua.</small>
</div>`
        expect(html).toMatchSnapshot()
    })

    it('Does not match snapshot', () => {
        expect('Anim ea ea velit aliqua quis Lorem dolor.').toMatchSnapshot()
    })

    it('Does not match HTML snapshot', () => {
        const html = `<div>
  <h2>Hello, there!</h2>
  <p class="wrong">Ad do ipsum anim excepteur aute commodo quis ullamco.</p>
  <sub>Culpa elit minim non ut velit mollit ullamco ea.</sub>
</div>`
        expect(html).toMatchSnapshot()
    })
})
