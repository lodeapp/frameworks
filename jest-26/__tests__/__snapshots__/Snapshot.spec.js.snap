// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A test with snapshots Does not match HTML snapshot 1`] = `
"<div>
  <h2>Hello world!</h2>
  <p>Esse excepteur aliqua proident aliquip deserunt fugiat officia et consequat sit sint aliqua exercitation.</p>
  <small>Culpa cupidatat ut velit aliqua.</small>
</div>"
`;

exports[`A test with snapshots Does not match snapshot 1`] = `"Nisi occaecat ullamco laborum laborum voluptate consectetur do minim et officia enim."`;

exports[`A test with snapshots Matches HTML snapshot 1`] = `
"<div>
  <h2>Hello world!</h2>
  <p>Esse excepteur aliqua proident aliquip deserunt fugiat officia et consequat sit sint aliqua exercitation.</p>
  <small>Culpa cupidatat ut velit aliqua.</small>
</div>"
`;

exports[`A test with snapshots Matches snapshot 1`] = `"Velit dolore laboris qui est."`;
